src/webhooks/pubsub/gocdPausedPipelineBot.ts

```diff
@@ -0,0 +1,135 @@
+import * as Sentry from '@sentry/node';
+import moment from 'moment-timezone';
+
+import { GitHubOrg } from '@/api/github/org';
+import { fetchDashboard } from '@/api/gocd';
+import { bolt } from '@/api/slack';
+import {
+  GETSENTRY_ORG,
+  GOCD_ORIGIN,
+  GOCD_PAUSED_PIPELINE_REMINDERS,
+} from '@/config';
+import { GoCDDashboardResponse } from '@/types';
+
+const GOCD_PAUSED_PIPELINE_REMINDER_TEXT_SINGULAR =
+  'A pipeline has been paused for an extended period of time. If this is unintentional, please look into unpausing it if it is safe to do so.';
+const GOCD_PAUSED_PIPELINE_REMINDER_TEXT_PLURAL =
+  'Multiple pipelines have been paused for an extended period of time. If this is unintentional, please look into unpausing them if it is safe to do so.';
+
+type PausedPipelineInfo = {
+  pipelineName: string;
+  durationPaused: moment.Duration;
+};
+
+export const triggerPausedPipelineBot = async (
+  org: GitHubOrg,
+  now: moment.Moment
+) => {
+  if (org !== GETSENTRY_ORG) {
+    return;
+  }
+  let dashboardResult: GoCDDashboardResponse;
+  try {
+    dashboardResult = await fetchDashboard();
+  } catch (err) {
+    Sentry.captureException(err);
+    return;
+  }
+  const remindersByChannel = getRemindersByChannel(dashboardResult, now);
+  for (const [channel, pausedPipelineInfos] of remindersByChannel.entries()) {
+    const pausedPipelineReminderText = getReminderText(pausedPipelineInfos);
+    postMessageToSlack(
```

Comment:
Should we use await/async here? To ensure behavior is consistent with what we want and nothing appears out of order for any reason

Reply:
Good catch